src/modules/taxonomy/resources/index.ts

```diff
@@ -76,29 +107,37 @@ export async function getResourceId({ id, language }) {
   return resourceId;
 }
 
-export function queryResources(contentId, language, contentType = 'article') {
-  return fetchAuthorized(
+export const queryResources = async (
+  contentId: string,
+  language: string,
+  contentType = 'article',
+): Promise<Resource[]> => {
+  return await fetchAuthorized(
```

Comment:
Når du bruker `.then()` syntaksen, trenger du ikke å ha med `async/await` i funksjonssignaturen.   
Å bruke `return await a.then(b)` fra en `async` funksjon er litt dårlig pattern. Grunnen til det er at `async` funksjoner returnerer alltid en Promise, samt `.then()` returnerer også en Promise, så sluttresultatet fra denne funksjonen blir uansett en Promise, uavhengig om du awaiter resultate eller ikke. Det du ender opp med når du sier `return await` er at du eksplisitt blokkerer funksjonen fra å fullføre, for å ut en verdi, som uansett blir wrappa i en Promise.

Jeg fikk satt opp en liten POC for å teste om det er noe forskjell mellom de to implementasjonene, og ble overrasket over resultatene. Funksjonen med `async/await`:

![image](https://user-images.githubusercontent.com/10486712/123674975-840fc800-d842-11eb-92b0-01eb7b11995f.png)

Funksjonen uten `async/await`:

![image](https://user-images.githubusercontent.com/10486712/123675467-0b5d3b80-d843-11eb-98f9-af1ec15b69fd.png)

Over 300ms i forskjell, siden funksjonen må vente på at Promisen resolver. Koden som jeg brukte til å måle:
```tsx
 // Med async/await (~300ms)
    const testAsync = async () => {
    const start = performance.now();
    await getArticle(id);
    const end = performance.now();
    console.log("TIME: ", end - start);
  }
// uten async/await (~1ms)
    const testAsync = () => {
    const start = performance.now();
    getArticle(id);
    const end = performance.now();
    console.log("TIME: ", end - start);
  }
```
getArticle(id) returnerer en Promise fra `resolveJsonOrRejectWithError` funksjonen. Brukte ulike id'er for hver gang for at cache ikke skulle påvirke resultater.

Konklusjonen er at du kan fjerne `async/await` syntakt fra endepunktene og bare returnere 
```tsx
=> fetchAuthorized(asd).then(resolveJson);
```